validator/client/validation_client.go

```diff
@@ -84,6 +100,7 @@ func (c *ValidationClient) Start(ctx_in context.Context) error {
 	c.room.Store(int32(room))
 	c.wasmModuleRoots = moduleRoots
 	c.name = name
+	c.stylusArchs = stylusArchs
```

Comment:
I think this code should be in an Initialize function. That change might belong in a separate PR though. As-is, I think there's a race condition with e.g. StylusArchs, where Started might return true before Start is actually finished.

Reply:
Good point. Creating a separate issue for that.

Reply:
On second thought - there is a simple fix of the possible for now. We may go for Initialize later.